CONTRIBUTING.md

```diff
@@ -1,33 +1,35 @@
 # Contributing to TestCafe
 
-TestCafe would not be possible without active support from the community. We certainly appreciate and would
-encourage your contributions, no matter how large or small.
+TestCafe would not be possible without active support from the community. We appreciate and encourage your contributions, no matter how big  or small.
 
-To that end, please review our contribution guidelines first.
+Review our contribution guidelines:
 
 * [Code of Conduct](#code-of-conduct)
 * [General Discussion](#general-discussion)
 * [Reporting a Problem](#reporting-a-problem)
 * [Code Contribution](#code-contribution)
+* [Contribute to Documentation](#contribute-to-documentation)
+  * [Documentation for the Current Functionality](#documentation-for-the-current-functionality)
+  * [Documentation for New Features](#documentation-for-new-features)
 
 ## Code of Conduct
 
-TestCafe has adopted a [Contributor Code of Conduct](CODE_OF_CONDUCT.md), abide by its terms.
+TestCafe abides by the [Contributor Code of Conduct](CODE_OF_CONDUCT.md).
 
 ## General Discussion
 
 Join the TestCafe community on Stack Overflow: ask and answer [questions with the TestCafe tag](https://stackoverflow.com/questions/tagged/testcafe).
 
 ## Reporting a Problem
 
-If you find a problem when using TestCafe, please file an issue in our [GitHub repository](https://github.com/DevExpress/testcafe/issues).
+If you encounter a bug when using TestCafe, please file an issue in our [GitHub repository](https://github.com/DevExpress/testcafe/issues).
 However, to save some time, please search through the existing issues to see if the problem has already been reported or addressed.
 
 When you create a new issue, template text is automatically added to its body. To help us understand the issue you're describing, be sure to fill in all sections in this template. We process issues with insufficient details after all the others, which may take significant time without any guarantees.
 
 ## Code Contribution
 
-Please follow the steps below when submitting your code.
+Follow the steps below when submitting your code.
 
 1. Search the [list of issues](https://github.com/DevExpress/testcafe/issues) to see if there
   is an issue for the bug or feature you are going to work on. If you do not find one, please create your own.
```

Comment:
"Search the [list of issues](https://github.com/DevExpress/testcafe/issues) to see if there
  is an issue for the bug or feature you are going to work on. If you do not find one, please create your own." -> Search the [list of issues](https://github.com/DevExpress/testcafe/issues) to see if there is an issue for the bug or feature you are going to work on or create a new one.